server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/TwoTierCompactionStrategy.java

```diff
@@ -61,6 +64,18 @@ public void verifyRequiredProperties(Object... objectsToVerify) throws IllegalAr
     }
   }
 
+  /**
+   * Calculates the total size of input files in the compaction plan
+   */
+  private Long calculateTotalSize(MajorCompactionRequest request, CompactionPlan plan) {
+    long totalSize = 0;
+    Map<FileRef,DataFileValue> allFiles = request.getFiles();
+    for (FileRef fileRef : plan.inputFiles) {
```

Comment:
I am wondering if you could iterate over `request.getFiles().values()`?  Are the keys of request.getFiles() and plan.inputFiles the same?  I am not sure, would have to look at the code.